source/feature/remove-snapshot.html.md

```diff
@@ -0,0 +1,116 @@
+
```

Comment:
The file should be located in 'source/develop/release-management/features/storage'